dashboard/models.py

```diff
@@ -185,14 +218,14 @@ def course_date_range(self):
             start = self.term.date_start
         else:
             logger.warning("No date_start value was found for course or term; setting to current date and time")
-            start = datetime.now(pytz.UTC)
+            start = datetime.datetime.now(pytz.UTC)
```

Comment:
NitPick: I feel `datetime.datatime` can have a better import like `from datetime import datetime`. This is something going over all the file. But next time somebody touch this should fix it

Reply:
I tend to agree, but it's not a huge deal. I prefer to import as `from datetime import datetime, timedelta`.

Reply:
I can update this, I feel like there was some reason I wasn't but I don't remember what it was.